### PR TITLE
Inject every time step

### PR DESCRIPTION
**Pull request by drichte2**
Modified a few things to make it straightforward to inject droplets every time step. Do this by setting "droplet_inject_elapse" to a negative number in the namelist file. This is good for the ASD runs, but for other applications it might be necessary to wait longer than a single time step to inject.

### CODE REVIEW

File: src/solve1.F [hunk 1]
```diff
@@ -416,6 +416,7 @@ subroutine solve1(nstep,rbufsz,num_soil_layers,                  &
       else
         tqnudge = .false.
       endif
+      tqnudge = .false.
```
Comment (johnmauff) [inline src/solve1.F]: @drichte2 This seems problematic to turn tqnudge off by default.  Is this valid for all uses of CM1 or just for CM1 running testcase == 10 and with droplets?

Comment (sjsprecious) [inline src/solve1.F]: So this setting will always set `tqnudge` to `False`. Is this intended? If so, we probably should remove the `if` block above.

**Review by johnmauff — COMMENTED** [review]
We need to have to correctness verification results.

**Review by sjsprecious — COMMENTED** [review]
Thanks @drichte2 for submitting this PR. I have a minor clarification question before I run the verification test.

**Comment by drichte2** [comment]
This was a manual thing I did to configure the ASD runs like I wanted them. `testcase==10` turns on nudging by default, which is what I am overriding here. I suppose one way is to define a new `testcase` number which is identical except for this point, but this seemed easier for these specific runs.

I'll take it out of the PR and only use it in my local version

**Comment by sjsprecious** [comment]
The verification results are similar to the previous PR (Derecho; CPU: ifort, 2 MPI ranks; GPU: nvhpc, 2 MPI ranks + 2 A100 GPUs):

# Metrics
38 stat variables are identical.
41 stat variables show a mean relative difference > 1e-06
7 stat variables show a mean relative difference <= 1e-06

# 0th-order diagnostics
4 stat variables are identical.
8 stat variables show a mean relative difference > 1e-06
0 stat variables show a mean relative difference <= 1e-06

# 1st-order diagnostics
254 stat variables are identical.
2 stat variables show a mean relative difference > 1e-06
0 stat variables show a mean relative difference <= 1e-06

**Review by sjsprecious — APPROVED** [review]
Looks good to me.

**Comment by johnmauff** [comment]
Thanks for the changes.  They look good to me.

**Pull request merged**